index.html

```diff
@@ -3,22 +3,225 @@
+                    <p class="footer__text">At GoodCall, we believe that the best decisions are backed by data. Moreover, we believe that data should be free, transparent and easy to understand. We aim to help students and their families make the best decisions at critical moments – for college, careers and beyond.</p>
+                </div>
+                <div class="columns large-2">
+                    <p class="footer__heading">GoodCall</p>
+                    <ul>
+                        <li><a href="#">About GoodCall</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Press</a></li>
+                        <li><a href="#">Contact Us</a></li>
+                    </ul>
+                </div>
+                <div class="columns large-1">
+                    <p class="footer__heading">Social</p>
+                    <ul>
+                        <li><a href="#">Twitter</a></li>
+                        <li><a href="#">Facebook</a></li>
+                        <li><a href="#">LinkedIn</a></li>
+                        <li><a href="#">Youtube</a></li>
+                        <li><a href="#">Google+</a></li>
+                    </ul>
+                </div>
+                <div class="columns large-2">
+                    <p class="footer__heading">Legal</p>
+                    <ul>
+                        <li><a href="#">Privacy Policy</a></li>
+                        <li><a href="#">Terms of Use</a></li>
+                    </ul>
+                </div>
+                <div class="columns large-2">
+                    <p class="footer__heading">Location</p>
+                    <ul>
+                        <li><a href="#">GoodCall, LLC</a></li>
+                        <li><a href="#">1101 Red Ventures Dr Fort Mill, SC 29707</a></li>
+                    </ul>
+                </div>
+            </div>
+        </footer>
 
-        <script src="node_modules/jquery/dist/jquery.min.js"></script>
+        <div id="login-modal" class="reveal-modal  modal" data-reveal aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
+          <h2 class="modal__title" ">Log-In</h2>
+          <p class="modal__error">Please enter a valid email address.</p>
+          <label class="modal__label"><span class="email">Email</span></label>
+          <input type="email" class="modal__email" placeholder="Email or Username">
+          <label class="modal__label">Password</label>
+          <input type="password" class="modal__password" placeholder="Password">
+          <a href="#" class="button">Log In</a>
+          <p class="modal__account">Don't have an account? <span class="sign-up">Sign-Up Now</span></p>
+        </div>
+
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
         <script src="node_modules/zurb-foundation-5/js/foundation/foundation.js"></script>
         <script src="assets/js/project.js"></script>
```

Comment:
I would put your project.js file at the end of all your includes, after foundation and slick.  That way you don't end up trying to use something that doesn't exist on the page because it hasn't loaded yet.